packages/yew/Cargo.toml

```diff
@@ -26,13 +26,14 @@ slab = "0.4"
 wasm-bindgen = "0.2"
 yew-macro = { version = "^0.19.0", path = "../yew-macro" }
 thiserror = "1.0"
-
 futures = { version = "0.3", optional = true }
 html-escape = { version = "0.2.9", optional = true }
 implicit-clone = { version = "0.2", features = ["map"] }
 base64ct = { version = "1.5.0", features = ["std"], optional = true }
 bincode = { version = "1.3.3", optional = true }
 serde = { version = "1", features = ["derive"] }
+tokio = { version = "1.19", features = ["sync"] }
+tokio-stream = { version = "0.1.9", features = ["sync"] }
```

Comment:
Shouldn't tokio still be an optional dependency on client-side apps?

Reply:
`tokio::sync::mpsc` and `tokio::sync::oneshot` are exposed as part of `yew::platform::sync`.

We are using these task synchronisation primitives internally.
This gives users the choice to opt for the same implementation that Yew uses.

As we are making a pseudo-runtime, we should also (eventually) provide items that a normal runtime also provides so users don't have to constantly use `#[cfg]` to select implementation that works with the current backend Yew is using. (e.g.: `gloo::timers` vs `tokio::time`)

`tokio` variant is always chosen for task synchronisation primitives as it provides a slightly better implementation.